### PR TITLE
fix: error from production environment

### PR DESCRIPTION
**Pull request by tobiichi3227**
In this PR, we collected the following errors and fixed them.
- Clicking any link will reload page, which makes our page too slow
- The acct-page displayed the content of the `<script></script>`
- The proclass is not sorted
- The proclass info dialog makes the background gray and prevents the user from interacting when they click any link in this dialog

### CODE REVIEW

File: src/handlers/manage/pro.py [hunk 1]
```diff
@@ -491,7 +533,7 @@ async def post(self, page=None):
 
                 await ProService.inst.update_test_config(pro_id, pro['testm_conf'])
                 await LogService.inst.add_log(
-                                        f'{self.acct.name} had been send a request to delete a single file:{filename} of the problem #{pro_id}',
+                    f'{self.acct.name} had been send a request to delete a single file:{filename} of the problem #{pro_id}',
```
Comment (lifeadventurer) [inline src/handlers/manage/pro.py]: - Change "had been send" to "has sent" to reflect the correct tense and form.
- Change "of the problem" to "for problem" for better clarity.

File: src/static/index.js [hunk 2]
```diff
@@ -125,11 +125,12 @@ var index = new function() {
...
         $(document).on('click', 'a', function(e) {
             let cur_href = location.href;
+
             let href = $(this).attr('href');
+            let target = $(this).attr('target');
             if (href == undefined || href.length == 0) return;
-            if ($(this).attr('target') !== "") {
-                return;
-            }
+            if (target != undefined && target != "") return
```
Comment (lifeadventurer) [inline src/static/index.js]: If you want to check if `target` is truthy (not `undefined`, `null`, `""`, `0`, `NaN`, or `false`), you can simply do:
```javascript
if (target) return;
```

**Review by lifeadventurer — CHANGES_REQUESTED** [review]
Overall, it's great to see that you've finally provided a description for this pull request! This is a significant improvement, as previous submissions often had a blank description. Clear documentation helps everyone understand the changes being made and facilitates smoother collaboration. Keep up the good work!

File: src/static/index.js (hunk 2 above)
Comment (tobiichi3227) [reply src/static/index.js]: Great!
Too many types in js represent `false`. :rofl:

**Review by lifeadventurer — APPROVED** [review]

**Pull request merged**